misc/packaging/apache/wsgi.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/wsgi.py b/wsgi.py
+index d66f9c5..86615de 100644
```

Comment:
What do we need this patch for? Isn't it part of common wsgi.py?

Reply:
For the apache and gunicorn the ansible_runner_service.py is used outside of the project.
When installing I moved it to the runner_service folder. But some parts of the project still use the original type.

Reply:
My bad accidentally kept the change when testing